src/main/resources/jenkins/security/plugins/ldap/Messages.properties

```diff
@@ -1,2 +1,50 @@
-FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
\ No newline at end of file
+FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
+LDAPSecurityRealm.LoginHeader=Login
+LDAPSecurityRealm.AuthenticationSuccessful=Authentication: successful
+LDAPSecurityRealm.AuthenticationFailed=Authentication: failed for user "{0}"
+LDAPSecurityRealm.AuthenticationFailedEmptyPass=Authentication: failed for user "{0}" with empty password
+LDAPSecurityRealm.UserId=User ID: {0}
+LDAPSecurityRealm.UserDn=User Dn: {0}
+LDAPSecurityRealm.UserDisplayName=User Display Name: {0}
+LDAPSecurityRealm.UserEmail=User email: {0}
+LDAPSecurityRealm.GroupMembership=Group membership:
+LDAPSecurityRealm.LookupHeader=Lookup
+LDAPSecurityRealm.UserLookupSuccessful=User lookup: successful
+LDAPSecurityRealm.UserLookupFailed=User lookup: failed for user "{0}"
+LDAPSecurityRealm.UserLookupInconclusive=User lookup: user "{0}" may or may not exist.
+LDAPSecurityRealm.UserLookupDoesNotExist=User lookup: user "{0}" does not exist.
+LDAPSecurityRealm.UserLookupBadCredentials=User lookup: bad credentials for user lookup.
+LDAPSecurityRealm.GroupMembershipMatch=User groups consistent (login and lookup)
+LDAPSecurityRealm.GroupMembershipMismatch=User groups inconsistent (login versus lookup)
+LDAPSecurityRealm.GroupLookupSuccessful=Group lookup: successful ({0,choice,1#{0} group|1<{0,number,integer} groups})
+LDAPSecurityRealm.NoDisplayName=No display name specified for user!
+LDAPSecurityRealm.EmptyDisplayName=Retrieved display name was empty!
+LDAPSecurityRealm.CouldNotRetrieveDisplayName=Could not retrieve the display name attribute.
+LDAPSecurityRealm.NoEmailAddress=No email address specified for user!
+LDAPSecurityRealm.EmptyEmailAddress=Retrieved email address was empty!
+LDAPSecurityRealm.CouldNotRetrieveEmailAddress=Could not retrieve the email address attribute.
+LDAPSecurityRealm.IsAttributeNameCorrect=Is the attribute name "{0}" correct?
+LDAPSecurityRealm.AvailableAttributes=Available attributes are:
+LDAPSecurityRealm.UserLookupManagerDnRequired=Is a Manager DN and password required to lookup user details?
+LDAPSecurityRealm.UserLookupManagerDnCorrect=Is a Manager DN and password correct?
+LDAPSecurityRealm.UserLookupManagerDnPermissions=Does the Manager DN have permissions to perform user lookup?
+LDAPSecurityRealm.GroupLookupManagerDnPermissions=Does the Manager DN have permissions to perform group lookup?
+LDAPSecurityRealm.GroupLookupManagerDnRequired=Is a Manager DN and password required to lookup group details?
+LDAPSecurityRealm.UserLookupSettingsCorrect=Are the user search base and user search filter settings correct?
+LDAPSecurityRealm.GroupLookupSettingsCorrect=Are the group search base and group search filter settings correct?
+LDAPSecurityRealm.NoGroupMembership=No group membership reported
+LDAPSecurityRealm.BasicGroupMembership=No LDAP group membership reported.
```

Comment:
"reported" means "found"?
We think of  "No LDAP group membership found".

What's the difference with "LDAPSecurityRealm.NoGroupMembership"? what kind of other group is there? Do we mean that the user is not part of the "AUTHENTICATED" group?